components/Members/Skeleton/SkeletonCards.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import SkeletonCard from './SkeletonCard';
+
+const SkeletonCards = ({ cards = 6 }: { cards?: number }) => {
+  const cardsToRender = Array(cards)
+    .fill(null)
+    .map((_, i) => i);
+  return (
+    <div className="flex mx-auto w-[90%] flex-wrap gap-4 mt-6 justify-between">
+      {cardsToRender.map(card => (
+        <SkeletonCard key={card} />
+      ))}
+    </div>
```

Comment:
This approach uses 2 `map` functions, we can use the power of `Array.from` [second argument](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#mapfn) to reduce one mapping function : 



```js
const SkeletonCards = ({ cards = 6 }: { cards?: number }) => {
  return (
    <div className="flex mx-auto w-[90%] flex-wrap gap-4 mt-6 justify-between">
      {Array.from({ length: cards }, (_, i) => (
        <SkeletonCard key={i} />
      ))}
    </div>
  );
};
```
Or if you want a more common (and maybe more understandable) approach you can create an array of components and render them, something like this : 


```js
const SkeletonCards = ({ cards = 6 }) => {
  const cardElements = [];
  for (let i = 0; i < cards; i++) {
    cardElements.push(<SkeletonCard key={i} />);
  }
  
  return (
    <div className="flex mx-auto w-[90%] flex-wrap gap-4 mt-6 justify-between">
      {cardElements}
    </div>
  );
};

Reply:
The approach I've used in my code is typescript safe AND for SMALL looping it doesn't matter to the performance